### PR TITLE
chore: improve date/time UDF descriptions

### PR DESCRIPTION
**Pull request by spena**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_
Make changes in the UDF date/time functions.

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/datetime/DateToString.java [hunk 1]
```diff
@@ -32,10 +32,11 @@
     name = "datetostring",
     category = FunctionCategory.DATE_TIME,
     author = KsqlConstants.CONFLUENT_AUTHOR,
-    description = "Converts an integer representing days since epoch to a date string"
+    description = "Converts a number of days since 1970-01-01 00:00:00 UTC/GMT to a date string"
```
Comment (JimGalasyn) [inline ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/datetime/DateToString.java:35]: "**the** number of days"?

File: ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/datetime/FromUnixTime.java [hunk 2]
```diff
@@ -24,11 +24,13 @@
 @UdfDescription(
     name = "from_unixtime",
     category = FunctionCategory.DATE_TIME,
-    description = "Converts a BIGINT millisecond timestamp value into a TIMESTAMP value.",
+    description = "Converts a number of millisecond 1970-01-01 00:00:00 UTC/GMT into "
```
Comment (JimGalasyn) [inline ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/datetime/FromUnixTime.java:27]: "the number of milliseconds"?

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a few suggestions.

**Review by jzaralim — APPROVED** [review]
LGTM, thank you!

**Pull request merged**